NthKey/Model/Configuration.storekit

```diff
@@ -0,0 +1,79 @@
+{
+  "products" : [
+
+  ],
+  "settings" : {
+    "_timeRate" : 0
+  },
+  "subscriptionGroups" : [
+    {
+      "id" : "5BC28C3C",
```

Comment:
Where do these identifiers come from?

Reply:
It set/appears automatically for Identifiable protocol support